### PR TITLE
Add cloudflare configuration flag to kocho.yml

### PR DESCRIPTION
**Pull request by zeisss**
We made cloudflare optional by default. Since the guide references cloudflare, we should activate it then ;)

### CODE REVIEW

**Comment by zeisss** [comment]
ping @puja108 @JosephSalisbury

**Comment by puja108** [comment]
hmm, I do like the default and that should also be what is the default documented. we should only add somewhere how to activate cloudflare in case you do want to use it.

**Comment by puja108** [comment]
should be good now, RFR @zeisss

File: docs/getting_started.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ of CoreOS clusters with customized versions of Etcd, Fleet and Docker.
 
 ## Prerequisites
 
- * Please have your AWS Access Keys ready
- * Please have CloudFlare Token ready
- * Please have `builder` installed
- * When running kocho in a docker container, docker needs to be installed
+ * Have your AWS Access Keys ready
+ * If you want to use [CloudFlare](https://www.cloudflare.com), have your CloudFlare Token ready
```
Comment (zeisss) [inline docs/getting_started.md]: lets move the optional stuff to the bottom of this list

**Comment by zeisss** [comment]
I would reorder the list, otherwise LGTM

File: docs/getting_started.md (hunk 1 above)
Comment (puja108) [inline docs/getting_started.md]: ack

**Pull request merged**